### PR TITLE
Use a dm-verity protected kata image and remove the initrd

### PR DESCRIPTION
**Pull request by dallasd1**
**What type of PR is this?**

**What this PR does / why we need it**:
Pull in kata-cc image instead of initrd to use dm-verity and reduce PSP measurement

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

**Review by miz060 — APPROVED** [review]

File: .pipelines/templates/.builder-release-template.yaml [hunk 1]
```diff
@@ -259,4 +259,4 @@ steps:
       vstsFeedPublish: 'CloudNativeCompute/AKS-Linux-VHD-Artifacts'
       vstsFeedPackagePublish: 'publishing-info-${{ parameters.artifactName }}'
       packagePublishDescription: 'VHD publishing info ${{ parameters.artifactName }}'
-      versionOption: minor
```
Comment (ms-mahuber) [inline .pipelines/templates/.builder-release-template.yaml:262]: minor nit: no unnecessary changes

**Review by ms-mahuber — APPROVED** [review]

File: .pipelines/templates/.builder-release-template.yaml (hunk 1 above)
Comment (dallasd1) [reply .pipelines/templates/.builder-release-template.yaml:262]: It shows the `versionOption` added back. I think this change might just be a false positive

Comment (ms-mahuber) [reply .pipelines/templates/.builder-release-template.yaml:262]: I think the newline was removed. anyways, let's get this merged.

**Review by ms-mahuber — APPROVED** [review]

**Review by miz060 — APPROVED** [review]
